app/backend/test/teet/integration/x_road_test.clj

```diff
@@ -184,8 +184,8 @@
                           property-registry/kinnistu-d-parse-response)
           omandiosad (:omandiosad parsed-resp)]
       (is (= :ok (:status parsed-resp)))
-      
-      
+
+
```

Comment:
Is this intentional?

Reply:
For some reason Ernos editor leaves these empty spaces the files sometimes and Idea just automatically removes them if I open the file.